ansible/roles/dashmate/tasks/main.yml

```diff
@@ -107,6 +107,16 @@
   ansible.builtin.set_fact:
     dashmate_config_version: "{{ (dashmate_version_result.stdout | split(' ') | first() | split('/'))[1] }}"
 
+- name: Check if default config is set
```

Comment:
Why do we need this?

Reply:
Because immediately after, we run `dashmate config get`. This command will fail on the initial run of deploy tool, because the default config is not yet set.